rt-forecast/functions/dates-to-epiweek.R

```diff
@@ -0,0 +1,25 @@
+library(tibble)
+library(dplyr)
+
+dates_to_epiweek <- function(df){
```

Comment:
nice I should steal the function :)

Reply:
its @kathsherratt - I stole it from the US forecasting hub. I think at some point we should make an epiforcasts utils package with these nice things in it that are always useful.